src/app_engine/probers.py

```diff
@@ -155,23 +187,23 @@ def create_collider_active_host(self, old_active_host, probing_results):
 
   def get(self):
     results = {}
-    for collider_instance in constants.WSS_HOST_PORT_PAIRS:
-      results[collider_instance] = self.probe_collider_instance(
-          collider_instance)
+    for instance in constants.WSS_INSTANCES:
+      host = instance[constants.WSS_INSTANCE_HOST_KEY]
+      results[host] = self.probe_collider_instance(instance)
     self.response.write(json.dumps(results, indent=2, sort_keys=True))
     self.store_instance_state(results)
 
   def probe_collider_instance(self, collider_instance):
-    url = 'https://' + collider_instance + '/status'
+    collider_host = collider_instance[constants.WSS_INSTANCE_HOST_KEY]
+    url = 'https://' + collider_host + '/status'
 
     error_message = None
     result = None
     try:
       result = urlfetch.fetch(
           url=url, method=urlfetch.GET, deadline=PROBER_FETCH_DEADLINE)
-    except Exception as e:
-      error_message = ('urlfetch throws exception: %s, url = %s'
-                       % (str(e), url))
+    except urlfetch.DeadlineExceededError as e:
```

Comment:
Is this the only exception we might get? Seems like there could be others we care about, certificate errors maybe?